src/Pages/RoomLists/Components/RoomInfo/RoomInfo.js

```diff
@@ -0,0 +1,187 @@
+import React, { useState, useEffect } from 'react';
+import StatusLabel from '../StatusLabel/StatusLabel';
+import styled from 'styled-components';
+import Theme from '../../../../Styles/Theme';
+
+const RoomInfo = () => {
+  const [roomInfoArr, setRoomInfoArr] = useState([]);
+
+  useEffect(() => {
+    fetch('/data/RoomListsData.json')
+      .then(res => res.json())
+      .then(data => {
+        setRoomInfoArr(data.results);
+      });
+  }, []);
+
+  // goToRoomDetail = id => {
+  //   history.push(`/rooms/${id}`);
+  // };
+
+  const countStars = rate => {
+    let rates = [];
+    for (let i = 0; i < rate / 2; i++) {
+      rates.push('★');
+    }
+    return <span>{rates}</span>;
+  };
+
+  const countComment = rate => {
+    if (rate >= 9) {
+      return <span>최고예요</span>;
+    }
+    if (rate >= 8) {
+      return <span>우수해요</span>;
+    }
+    if (rate >= 7) {
+      return <span>좋아요</span>;
+    }
+  };
```

Comment:
```suggestion
  const countComment = rate => {
    const commentMapper = {
      [rate >= 7]: "좋아요",
      [rate >= 8]: "우수해요",
      [rate >= 9]: "최고예요"
    }
    return <span>{commentMapper.true}</span>
  };
```
위와 같이도 가능합니다.
만약 if문을 사용할거라면 if, else if로 사용하는 것이 좋습니다.
```suggestion
  const countComment = rate => {
    let msg = ""
    if (rate >= 9) msg = "최고예요";
    else if (rate >= 8) msg = "우수해요";
    else if (rate >= 7) msg = "좋아요";
    
    return <span>{msg}</span>
  };
```

Reply:
다시 작성해보도록 하겠습니다!